src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.module.scss

```diff
@@ -0,0 +1,26 @@
+@import '~govuk-frontend/govuk/base';
+
+.downloadSection {
```

Comment:
These styles are duplicated in PublicationReleasePage.module.scss, is there a way to share them?

Reply:
I think I tried to do that and I hit some kind of problem - I'll investigate it again.

Reply:
Looking at it, I think the reason I didn't is due to the `<componentName>.module.scss`  naming scheme we have for styles related to components, and how there isn't an obvious place to put them in the explore-education-statistics-common project.

I've decided to put the common styles in _content.scss - not sure if that's an appropriate place for them or not.